### DISCUSSION TITLE
How are we going to keep <rh-footer> and elements that we don't own lined up?

### DISCUSSION THREAD

**Discussion by heyMP** [RedHat-UX/red-hat-design-system#846] [Ideas]
The idea of introducing global tokens for sharing state between multiple elements was brought up in this [<rh-footer> thread](https://github.com/RedHat-UX/red-hat-design-system/pull/841#discussion_r1147453873).  We have a situation in which we need to expose how large the spacing is between the viewport and the content so that the primary navigation, an element that we don't own, can keep it's contents lined up with footer.

It's easy when there is only one value, WE HAVE DESIGN TOKENS FOR THAT! 😀. But what if that value changes depending on different breakpoints? 🤔

![Screenshot 2023-03-24 at 8 14 11 AM](https://user-images.githubusercontent.com/3428964/227519336-13626c2c-889e-4289-bb7a-8fadc952b7a6.png)

**Comment by bennypowers** [comment]
long answers:
- pattern manifests
- own `<rh-navigation>` with DSD

short answers:
- docs and developer discipline, we can't fix the world during RC

**Reply by heyMP** [reply]
After RC of course :)
